test/unit/component/mxgraph/overlay/OverlayConverter.test.ts

```diff
@@ -14,29 +14,65 @@
  * limitations under the License.
  */
 import { OverlayConverter } from '../../../../../src/component/mxgraph/overlay/OverlayConverter';
-import { MxGraphCustomOverlayOptions } from '../../../../../src/component/mxgraph/overlay/custom-overlay';
+import { MxGraphCustomOverlayPosition } from '../../../../../src/component/mxgraph/overlay/custom-overlay';
 import { Overlay, OverlayPosition } from '../../../../../src/component/registry';
+import { StyleDefault } from '../../../../../src/component/mxgraph/StyleUtils';
 
 describe('overlay converter', () => {
   const overlayConverter = new OverlayConverter();
 
   it.each([
-    [<OverlayPosition>'top-left', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'left', verticalAlign: 'top' }],
-    [<OverlayPosition>'top-right', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'right', verticalAlign: 'top' }],
-    [<OverlayPosition>'top-center', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'center', verticalAlign: 'top' }],
-    [<OverlayPosition>'bottom-left', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'left', verticalAlign: 'bottom' }],
-    [<OverlayPosition>'bottom-right', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'right', verticalAlign: 'bottom' }],
-    [<OverlayPosition>'bottom-center', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'center', verticalAlign: 'bottom' }],
-    [<OverlayPosition>'middle-left', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'left', verticalAlign: 'middle' }],
-    [<OverlayPosition>'middle-right', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'right', verticalAlign: 'middle' }],
-    [<OverlayPosition>'start', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'left', verticalAlign: 'top' }],
-    [<OverlayPosition>'middle', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'center', verticalAlign: 'top' }],
-    [<OverlayPosition>'end', <MxGraphCustomOverlayOptions>{ horizontalAlign: 'right', verticalAlign: 'top' }],
-  ])('convert API overlay position %s to mxGraph overlay position %s', (apiPosition: OverlayPosition, mxGraphPosition: MxGraphCustomOverlayOptions) => {
+    [<OverlayPosition>'top-left', <MxGraphCustomOverlayPosition>{ horizontalAlign: 'left', verticalAlign: 'top' }],
```

Comment:
can be simplified: as the elements in the array are always the same, the types can be defined in the Array definition instead of being defined for each entries

Reply:
I didn't understand.
The elements are not always the same.